### PR TITLE
build the map feature

### LINKED ISSUE #8: Add location services
## Overview
As a user, I should be able to use GPS to display my location.

### PR DESCRIPTION
**Pull request by linhnk-ips**
- the app can access GPS to get the current location and show it on Google map

### CODE REVIEW

File: src/pwa/pages/maps.tsx [hunk 1]
```diff
@@ -1,6 +1,40 @@
...
+            setCurrentGeoPosition(position)
+        });
+    }
+
+    useEffect(() => {
+        getCurrentPosition();
+    }, []);
+
     return (
-        <h4>Map</h4>
```
Comment (uveta) [inline src/pwa/pages/maps.tsx:3]: I would add a small section above map, showing current GPS latitude and longitude.

File: src/pwa/pages/maps.tsx [hunk 2]
```diff
@@ -1,6 +1,40 @@
...
+        <div style={{ height: '100vh', width: '100%' }}>
+            {currentGeoPosition ?
+                <GoogleMapReact
```
Comment (uveta) [inline src/pwa/pages/maps.tsx]: Would it be possible to mark current location on the map?

**Review by uveta — CHANGES_REQUESTED** [review]

File: src/pwa/pages/maps.tsx (hunk 1 above)
Comment (linhnk-ips) [reply src/pwa/pages/maps.tsx:3]: I added the current location. help me verify it

File: src/pwa/pages/maps.tsx (hunk 2 above)
Comment (linhnk-ips) [reply src/pwa/pages/maps.tsx]: Yes, I add a masker as current location on the map

**Review by uveta — APPROVED** [review]

**Pull request merged**